ui/app/AppLayouts/Profile/Sections/AppearanceContainer.qml

```diff
@@ -286,22 +280,13 @@ ScrollView {
                 image.height: 128
                 //% "System"
                 control.text: qsTrId("system")
-                control.checked: profileModel.profile.appearance === AppearanceContainer.Theme.System
+                control.checked: globalSettings.theme === Universal.System
                 onRadioCheckedChanged: {
                     if (checked) {
-                        root.updateTheme(AppearanceContainer.Theme.System)
+                        root.updateTheme(Universal.System)
                     }
                 }
             }
-
-            // For the case where the theme was finally loaded by status-go in init(),
-            // update the theme in qml
-            Connections {
-                target: profileModel
-                onProfileChanged: {
-                    root.updateTheme(profileModel.profile.appearance)
-                }
-            }
```

Comment:
We're no longer interested in the setting coming from status-go, so we can get rid off this connection here entirely